timemachine/datasets/utils.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+from typing import List
+
+from rdkit import Chem
+
+import timemachine
+
+PATH_TO_FREESOLV = str(Path(timemachine.__file__).parent.parent / "timemachine/datasets/freesolv/freesolv.sdf")
```

Comment:
Nit: Redundant to do `parent.parent` then reattach `timemachine`

Reply:
Was intentional, so that path strings are relative to project root (can be more easily found / changed later if we do another reorganization), but probably makes more sense for it to be relative to package root?

Applied in https://github.com/proteneer/timemachine/pull/619/commits/f2401a5b5e4e2a3bf790f0146e36c0f6de070322

Reply:
Superseded by https://github.com/proteneer/timemachine/pull/619/commits/0c434fe4d71fec2f8226e45f22edac5cd413f853